core/domain/feedback_services.py

```diff
@@ -302,6 +302,48 @@ def get_message(thread_id, message_id):
         feedback_models.GeneralFeedbackMessageModel.get(thread_id, message_id))
 
 
+# TODO(anubhavsinha98): Remove this function once the message field in all
+# the threads are populated.
+def get_message_count(thread_id):
+    """Fetches the messages count of the given thread.
+
+    Args:
+        thread_id: str. The ID of the thread to count messages in.
+
+    Returns:
+        int. The count of messages of the given thread.
+    """
+    return feedback_models.GeneralFeedbackMessageModel.get_message_count(
+        thread_id)
+
+
+# TODO(anubhavsinha98): Remove this function once the message field in all
+# the threads are populated.
+def thread_message_count(thread_id):
```

Comment:
I don't think you should add this function at all generally to the main codebase, since it is just a helper function used in tests. Have the tests do the logic in this function directly.

Reply:
I will add these functions in the test class.

Reply:
Added in the test class.

Reply:
DONE